src/activities/ActivityUsageEntity.js

```diff
@@ -68,21 +68,36 @@ export class ActivityUsageEntity extends Entity {
 
 	canEditDueDate() {
 		const dueDate = this._getDueDateSubEntity();
-		return (dueDate && dueDate.hasActionByName(Actions.activities.update)) || false;
-	}
-
-	saveDueDateAction() {
-		const dueDate = this._getDueDateSubEntity();
-		return dueDate && dueDate.getActionByName(Actions.activities.update);
+		return (dueDate && dueDate.hasActionByName(Actions.activities.update))
+			|| this._entity.hasActionByName(Actions.activities.startAddNew)
+			|| false;
```

Comment:
What's the purpose of `|| false`? Wouldn't the statement just be false if the other components are both false?

Reply:
It's just type coercion due to the name of the method - since it's `can*`, I infer that it should return `true` or `false` specifically, rather than `undefined` (which it would without the `|| false`)

Reply:
...except that it calls `hasActionByName` which pretty sure returns a boolean anyway. So nvm, this is not required and I'll remove it!